packages/app/src/contact-support/containers/contact-support-flow.container.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { connect } from "react-redux"
+import { ReduxRootState, RootState, TmpDispatch } from "Renderer/store"
+import ContactSupportFlow from "App/contact-support/components/contact-support-flow.component"
+import { files } from "App/contact-support/reducers"
+import {
+  sendTicket,
+  SendTicketPayload,
+} from "App/contact-support/actions/send-ticket.action"
+import { closeContactSupportFlow } from "App/contact-support/actions/close-contact-support-flow.action"
+
+const mapStateToProps = (state: RootState & ReduxRootState) => ({
+  files: files,
+  state: state.contactSupport.state,
+})
+
+const mapDispatchToProps = (dispatch: TmpDispatch) => ({
```

Comment:
You can use simpler construction here
```ts
const mapDispatchToProps = {
    closeContactSupportFlow, 
    sendTicket,
}
```

Reply:
https://github.com/mudita/mudita-center/pull/561/commits/2eee9b5cab017bb9d533a4b3b3e2138cf475ded6